src/fvm_multicell.hpp

```diff
@@ -592,7 +625,8 @@ void fvm_multicell<Backend>::initialize(
     std::vector<size_type> group_parent_index(ncomp);
 
     // setup per-cell event stores.
-    events_ = util::make_unique<multi_event_stream>(ncell_);
+    events_ = util::make_unique<deliverable_event_stream>(ncell_);
```

Comment:
Hmm, probably not! I honestly cannot recall the motivation for making the original `multi_event_stream` a unique pointer. It might once have been living in the cell group instead of the lowered cell?

I'll make them simple member variables, and make sure nothing awful happens :)